react/MuiCozyTheme/Dialog/DialogCloseButton.jsx

```diff
@@ -1,3 +1,14 @@
-import ModalCross from '../../Modal/ModalCross'
+import React from 'react'
+import DialogCloseButton from '../../Dialog/DialogCloseButton'
+import createDepreciationLogger from '../../helpers/createDepreciationLogger'
 
-export default ModalCross
+const logDepecratedMUIDialog = createDepreciationLogger()
+
+const OldDialogCloseButton = props => {
```

Comment:
NIt : I'm not very comfortable with 'old' or 'new' keywords, maybe we should call it `DeprecatedDialogCloseButton` ?